lib/clis/wolkenkit/health/healthCommand.ts

```diff
@@ -0,0 +1,88 @@
+import { buntstift } from 'buntstift';
+import { Client } from '../../../apis/getHealth/http/v2/Client';
+import { Command } from 'command-line-interface';
+import { HealthOptions } from './HealthOptions';
+import { validatePort } from '../dev/validatePort';
+
+const healthCommand = function (): Command<HealthOptions> {
+  return {
+    name: 'health',
+    description: 'Run a health check against a process.',
+
+    optionDefinitions: [
+      {
+        name: 'protocol',
+        description: 'set the protocol',
+        parameterName: 'protocol',
+        type: 'string',
+        isRequired: false,
+        defaultValue: 'http'
+      },
+      {
+        name: 'host-name',
+        description: 'set the host name of the server',
+        parameterName: 'hostName',
+        type: 'string',
+        isRequired: false,
+        defaultValue: 'localhost'
+      },
+      {
+        name: 'health-port',
+        alias: 'e',
+        description: 'set a port for the health endpoint',
+        parameterName: 'port',
+        type: 'number',
+        isRequired: false,
+        defaultValue: 3001,
+        validate: validatePort
+      },
+      {
+        name: 'base-path',
+        description: 'set the base path of the health api',
```

Comment:
```suggestion
        description: 'set the health base path',
```

(To be consistent with the other descriptions)